### ISSUE TITLE
I want to solve overfitting.

### ISSUE THREAD

**Issue by cleamm** [autonomousvision/stylegan_xl#82]
hello. I am a student. Inspired by the projected gan, I used it and I want to try this model. Unlike previous models, overfitting occurs. How can I solve this? Should I use dropout? If there is a solution or some other way to solve it, please help!

**Comment by nom57** [comment]
I am having similar issues and its clearly visible in stats.jsonl that the discriminator is getting lower and lower loss score.
I think this means diffaug is broken and it just lets the discriminator overfit.
augmentation also always stays = 0.000 in logs / the printed training information

is this a bug or its intended to work like this ?

**Comment by nom57** [comment]
also @cleamm please share your stats.jsonl content so I can compare it to mine as I debugged my issues a lot with SGXL and it was always dead giveaway what is wrong.
